<launch>
    <node name="pose_finder" type="pose_finder.py" pkg="ur_cart_move" output="screen">
        <param name="test_prefix" value="/sim2" />
        <param name="tf_prefix" value="/sim1" />
        <remap from="/joint_states" to="/sim1/joint_states" />
        <remap from="/ur_joint_states" to="/sim1/ur_joint_states" />
        <remap from="/ur_mode_states" to="/sim1/ur_mode_states" />
        <remap from="/ur_joint_command" to="/sim1/ur_joint_command" />
        <remap from="/ur_mode_command" to="/sim1/ur_mode_command" />
    </node>
</launch>
